src/components/Table/GridTable.tsx

```diff
@@ -1048,10 +1056,17 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
 
   let currentColspan = 1;
 
+  let firstContentColumnStylesApplied = false;
+
   const rowNode = (
     <Row css={rowCss} {...others} data-gridrow {...getCount(row.id)}>
       {columns.map((column, columnIndex) => {
-        const { wrapAction = true } = column;
+        const { wrapAction = true, isAction = false } = column;
+
+        const applyFirstContentColumnStyles = !isAction && !firstContentColumnStylesApplied;
+        if (!firstContentColumnStylesApplied && applyFirstContentColumnStyles) {
+          firstContentColumnStylesApplied = true;
+        }
```

Comment:
This might be too cute, but I think:

```
firstContentColumnStyledApplied ||= applyFirstContentColumnStyles;
```

Would replace the `if` statement

Reply:
Nice! I don't think I've ever used that operator.